porting/examples/linux_blemesh/include/syscfg/syscfg.h

```diff
@@ -1021,6 +1021,10 @@
 #define MYNEWT_VAL_BLE_MESH_TX_SEG_MSG_COUNT (4)
 #endif
 
+#ifndef MYNEWT_VAL_BLE_MESH_TX_SEG_RETRANSMIT_ATTEMPTS
```

Comment:
linux_blemesh does not build because define is incorrect here. There is no "TX" in the syscfg definition.

Reply:
Updated. Please review.